### PR TITLE
KEv2 cluster import fix (integrate changes)

### PR DESCRIPTION
**Pull request by a-blender**
Address https://github.com/rancher/rancher/issues/36128.

Integrate updates to KEv2 operators into the terraform provider

Fixes
- https://github.com/rancher/eks-operator/pull/59
- https://github.com/rancher/aks-operator/pull/52
- https://github.com/rancher/gke-operator/pull/73

### CODE REVIEW

File: go.mod
Comment (thedadams) [inline go.mod]: The operators need to be updated in Rancher first. Then, in Rancher, run `go generate`.

After that has been merged in Rancher, then you can update the commits here.

**Review by thedadams — CHANGES_REQUESTED** [review]

File: go.mod
Comment (a-blender) [reply go.mod]: Done! Waiting on a review for https://github.com/rancher/rancher/pull/36461

**Review by thedadams — CHANGES_REQUESTED** [review]
This PR needs to be updated once the Rancher PR is merged.

File: go.mod
Comment (a-blender) [reply go.mod]: Merged. Updating the commits now...

Comment (a-blender) [reply go.mod]: I updated the terraform provider commits with the top hash from `rancher/rancher:release/v2.6`, which is my merge commit for https://github.com/rancher/rancher/pull/36461.

Let me know if that's correct!

**Comment by thedadams** [comment]
Drone isn't running for some reason. The PR looks good, but we should make sure that the tests all pass before merging.

**Review by thedadams — APPROVED** [review]

**Review by jakefhyde — APPROVED** [review]

**Pull request merged**
